arroyo-sql/src/types.rs

```diff
@@ -241,6 +241,26 @@ impl StructDef {
         let schema_data_impl = if self.name.is_none() {
             // generate a SchemaData impl but only for generated types
             let name = self.struct_name();
+
+            let to_raw_string = if self.fields.len() == 1
+                && matches!(
+                    self.fields[0].data_type,
+                    TypeDef::DataType(DataType::Utf8, _)
+                ) {
+                let field = &self.fields[0].field_ident();
+                if self.fields[0].nullable() {
+                    quote! {
+                        self.#field.as_ref().map(|v| v.as_bytes().to_vec())
+                    }
+                } else {
+                    quote! {
+                        self.#field.as_bytes().to_vec()
```

Comment:
I think to_raw_string() is expecting an `Option<Vec<u8>>`

Reply:
Good catch, fixed and added a test